### PR TITLE
New alert messages

### PR DESCRIPTION
**Pull request by aaronchongth**
### Implemented feature

New messages that allow custom interactions to arise from the robots, fleets or other parts of the system, namely alerts, notifications, user interactions, etc

* `Alert`
* `AlertParameter`
* `AlertResponse`

### Example usage

This can hopefully extend to many more cases, other than these 3 examples provided 

Case 1
* Fleet adapters with custom `perform_actions` will require user input or responses for certain decisions, can publish an alert with all the possible responses listed
* API server picks it up and prompts the users on the frontend
* User chooses a response
* API server will publish the response back down to the fleet adapter

Case 2
* Fleet adapters wishes to inform users something has happened in the form of a notification (info,warning or error), with no user response required

Case 3
* Fleet adapters with custom `perform_actions` will require input or responses from other parts of the system (non-human), an alert can be sent out with it toggled not to be displayed
* The other system can respond using the response message as well

### CODE REVIEW

**Comment by xiyuoh** [comment]
LGTM, have tested it and it is comprehensive enough to contain the different types of information we would need to publish messages between the fleet adapter and RMF web.

**Review by xiyuoh — DISMISSED** [review]

File: rmf_task_msgs/msg/Alert.msg [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+uint8 TIER_INFO=0
+uint8 TIER_WARNING=1
+uint8 TIER_ERROR=2
+
+# Responses available for this alert. If no responses are expected
+# this field can be left empty
+string[] responses_available
+
+# Parameters that may be useful for custom interactions
+FleetAlertParameter[] alert_parameters
```
Comment (xiyuoh) [inline rmf_task_msgs/msg/Alert.msg:24]: This should be updated to `AlertParameter`

**Review by xiyuoh — CHANGES_REQUESTED** [review]

File: rmf_task_msgs/msg/Alert.msg (hunk 1 above)
Comment (aaronchongth) [reply rmf_task_msgs/msg/Alert.msg]: Yup, forgot to push my changes, thanks, https://github.com/open-rmf/rmf_internal_msgs/pull/71/commits/68690540bf4e7dafb52007ca85913e48b88312c9

**Review by xiyuoh — APPROVED** [review]
LGTM

**Pull request merged**